projects/Mallard/src/screens/log-in.tsx

```diff
@@ -10,7 +10,15 @@ import { FormField } from 'src/hooks/use-form-field'
 import { LoginLayout } from 'src/components/login/login-layout'
 import { EmailInput, PasswordInput } from 'src/components/login/login-input'
 import { LoginButton } from 'src/components/login/login-button'
-import AppleSignInButton from 'src/components/apple-sign-in-button/apple-sign-in-buttom'
+
+import { Platform } from 'react-native'
+
+function canOSSupportAppleSignin(): boolean {
+    const osVersion = Number(String(Platform.Version).split('.')[0])
+    const osName = Platform.OS
```

Comment:
I think you could just use Platform.OS directly in the line below since `osName` is only used once